src/notifications/settings/initialization.ts

```diff
@@ -0,0 +1,250 @@
+import {
+  DEFAULT_ENABLED_TOPIC_SETTINGS,
+  NotificationRelationship,
+  NOTIFICATIONS_DEFAULT_CHAIN_ID,
+  WALLET_GROUPS_STORAGE_KEY,
+} from '@/notifications/settings/constants';
+import {
+  AddressWithRelationship,
+  WalletNotificationSettings,
+} from '@/notifications/settings/types';
+import {
+  getAllNotificationSettingsFromStorage,
+  notificationSettingsStorage,
+  setAllNotificationSettingsToStorage,
+} from '@/notifications/settings/storage';
+import { notificationsSubscription } from '@/redux/explorer';
+import store from '@/redux/store';
+import {
+  subscribeWalletToAllEnabledTopics,
+  unsubscribeWalletFromAllNotificationTopics,
+} from '@/notifications/settings/firebase';
+import { InteractionManager } from 'react-native';
+import { logger, RainbowError } from '@/logger';
+import { removeNotificationSettingsForWallet } from '@/notifications/settings/settings';
+
+type InitializationStateType = {
+  alreadySaved: Map<
+    string,
+    { index: number; settings: WalletNotificationSettings }
+  >;
+  subscriptionQueue: WalletNotificationSettings[];
+  newSettings: WalletNotificationSettings[];
+};
+
+/**
+ Checks if group notification settings are present in storage
+ and adds default values for them if they do not exist.
+ */
+export const addDefaultNotificationGroupSettings = () => {
```

Comment:
Another/same migration?

Reply:
This is different and I think we could mive it to migrations

Reply:
Moved! But still leaving that in settings when resetting the local storage. If I get it correctly resetting local storage doesn't trigger migrations to run again right?